### PR TITLE
Theme Ultra

### PR DESCRIPTION
**Pull request by edgarcheverier**
Starts setup for ultra theme, adds the home, about, services and
portfolio pages, also adds sidebar for managing pages location.
News and Contact pages are still missing, after that
it's remaining to improve styles and clean the code.

theme-ultra

### CODE REVIEW

File: @factor/@themes/theme-ultra/src/page-home.vue [hunk 1]
```diff
@@ -0,0 +1,83 @@
+<template>
+  <div id="homeContainerID" class="homeContainer">
```
Comment (arpowers) [inline @factor/@themes/theme-ultra/src/page-home.vue]: We try and steer clear of using the ID attribute.

File: @factor/@themes/theme-ultra/src/page-home.vue [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    <div class="homeQuotesContainer">
```
Comment (arpowers) [inline @factor/@themes/theme-ultra/src/page-home.vue]: classnames should be kebab cased in core-factor. Style guide is needed on our part

File: @factor/@themes/theme-ultra/src/page-home.vue (hunk 1 above)
Comment (edgarcheverier) [reply @factor/@themes/theme-ultra/src/page-home.vue]: In my next PR I will change that, thanks!

File: @factor/@themes/theme-ultra/src/page-home.vue (hunk 2 above)
Comment (edgarcheverier) [reply @factor/@themes/theme-ultra/src/page-home.vue]: This change will require a bit more time but I will try to add it in my next PR, thanks!

**Pull request merged**